Source/JavaScriptCore/API/JSContextRefPrivate.h

```diff
@@ -94,6 +94,30 @@ JS_EXPORT void JSContextGroupSetExecutionTimeLimit(JSContextGroupRef group, doub
 */
 JS_EXPORT void JSContextGroupClearExecutionTimeLimit(JSContextGroupRef group) JSC_API_AVAILABLE(macos(10.6), ios(7.0));
 
+/*!
+@function
+@abstract Enables sampling profiler.
+@param group The JavaScript context group that the sampling profiler gets enabled.
```

Comment:
I would say "The JavaScript context group to start sampling"

Reply:
Fixed.